pkgs/development/libraries/noise-suppression-for-voice/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "noise-supression-for-voice";
+  version = "2020-10-12";
+
+  src = fetchgit {
+    url = "https://github.com/werman/noise-suppression-for-voice";
+    sha256 = "11pwisbcks7g0mdgcrrv49v3ci1l6m26bbb7f67xz4pr1hai5dwc";
+    rev = "15bac8f34018184d7d4de1b3b2ba98c433705f6c";
+  };
+
+  buildInputs = [ cmake ];
```

Comment:
https://nixos.org/manual/nixpkgs/stable/#reviewing-contributions-new-packages

`cmake` should be in `nativeBuildInputs` if it's a build-time only dependency. :slightly_smiling_face: